recipes/deodr/LICENSE

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
```

Comment:
I added a PR to address the manually packaged license file at https://github.com/martinResearch/DEODR/pull/34